src/Metadata/Resource/Factory/OperationDefaultsTrait.php

```diff
@@ -63,6 +63,10 @@ private function addGlobalDefaults(ApiResource|Operation $operation): ApiResourc
             $currentValue = $operation->{$getter}();
 
             if (\is_array($currentValue) && $currentValue) {
+                if ('Formats' === $upperKey && \is_string($value)) {
```

Comment:
```suggestion
                if (\is_string($value)) {
```

Reply:
IMO this should be the behavior for any defaults that needs to be merged with an array. Though in the end `default.formats` should only allow arrays.